state/nginx/configs/include/ssl.include

```diff
@@ -1,12 +1,12 @@
 ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
```

Comment:
@Adam-Ant I think you need to review your nginx TLS configs here. It's probably too permissive 
```suggestion
ssl_protocols TLSv1.2 TLSv1.3;
```

👉 https://cipherlist.eu/